internal/testutil/conduit.go

```diff
@@ -258,3 +271,88 @@ func SetupSecretConduit(t *testing.T) *v1alpha.Conduit {
 
 	return c
 }
+
+func SetupSourceProcConduit(t *testing.T) *v1alpha.Conduit {
+	t.Helper()
+	running := true
+
+	c := &v1alpha.Conduit{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "sample",
+			Namespace: "sample",
+		},
+		Spec: v1alpha.ConduitSpec{
+			Running:     &running,
+			Name:        "my-pipeline",
+			Version:     "v0.13.2",
+			Description: "my-description",
+			Registry: &v1alpha.SchemaRegistry{
+				URL: "http://apicurio:8080/apis/ccompat/v7",
+			},
+			Connectors: []*v1alpha.ConduitConnector{
+				{
+					Name:          "source-connector",
+					Type:          "source",
+					Plugin:        "builtin:generator",
+					PluginVersion: "latest",
+					PluginName:    "builtin:generator",
+					Settings: []v1alpha.SettingsVar{
+						{
+							Name:  "servers",
+							Value: "127.0.0.1",
+						},
+						{
+							Name:  "topics",
+							Value: "input-topic",
+						},
+					},
+					Processors: []*v1alpha.ConduitProcessor{
+						{
+							ID:           "proc1",
+							Name:         "proc1",
+							Plugin:       "builtin:base64.encode",
```

Comment:
minor, but if this processor is to represent a standalone processor (which I am guessing yes based on the URL being provided) we might want to reflect that in the plugin. 

```suggestion
							Plugin:       "standalone:foo.encode",
```